docs/index.rst

```diff
@@ -595,6 +678,50 @@ Users
        "last_name": "Watson"
      }
 
+.. http:post:: /password-resets/
```

Comment:
Ya, was wondering a bit about that. My thinking was `/confirmations/` could maybe be used for creating other kinds of confirmations in future. Maybe different endpoints for different kinds of confirmations makes more sense though, will make a new PR for this change.